linux/tray_manager_plugin.cc

```diff
@@ -4,7 +4,11 @@
 #include <gtk/gtk.h>
 #include <sys/utsname.h>
 
-#include <libappindicator/app-indicator.h>
+#ifdef HAVE_AYATANA
+#include <libayatana-appindicator/app-indicator.h>
+#else
+#include <appindicator/app-indicator.h>
```

Comment:
This should be #include <libappindicator/app-indicator.h> ?

Reply:
Good catch, thanks!